using-timescaledb/continuous-aggregates.md

```diff
@@ -0,0 +1,249 @@
+# Continuous Aggregates 
+TimescaleDB continuous aggregates allow users to speed aggregate queries which
+span large amounts of time by continuously calculating them in the background
+and materializing the aggregates in a separate view. Conntinuous aggregates use
+the background worker framework to run aggregates on a schedule as new data is
+inserted into a hypertable or when old data is modified. There's a bit more
+complexity to how they're implemented which you can read about in our [blog post
+announcing the feature][continuous-aggs-blog], but for now, suffice it to say
+that they can really help if you've been having trouble with long-running
+aggregate queries. 
+
+
+### Creating a Continuous Aggregate View [](create)
+Continuous aggregates are exposed to the user by extending the PostgreSQL
+[`CREATE VIEW`][postgres-createview] command using a `WITH` option, must contain
+a [`time_bucket`][time-bucket] function call in the group by clause and support
+all parallel-safe aggregates. 
+
+Let's suppose we have a hypertable `device_readings` created as so:
+```sql
+CREATE TABLE device_readings (
+      observation_time  TIMESTAMPTZ       NOT NULL,
+      device_id         TEXT              NOT NULL,
+      metric            DOUBLE PRECISION  NOT NULL,
+      PRIMARY KEY(observation_time, device_id)
+);
+SELECT create_hypertable('device_readings', 'observation_time');
+```
+
+And we want to query per-device readings aggregated on an hourly basis, we might
+create a continuous aggregate as so:
+
+```sql
+CREATE VIEW device_summary
+WITH (timescaledb.continuous) --This flag is what makes the view continuous
+AS
+SELECT
+  time_bucket('1 hour', observation_time) as bucket, --time_bucket is required
+  device_id,
+  avg(metric) as metric_avg, --We can use regular aggregates
+  max(metric)-min(metric) as metric_spread --We can also use expressions on aggregates and constants
+FROM
+  device_readings
+GROUP BY bucket, device_id; --We have to group by the bucket column, but can also add other group-by columns
+```
+This will create a continuous view called `device_summary` for the aggregation
+query, much like a normal view, but it also creates a materialization
```

Comment:
bold for "materialization hypertable" since it is the first time that term is introduced, and it's refrerenced heavily later.